### PR TITLE
[5/n] upgrade react-native 0.76

### PR DESCRIPTION
**Pull request by gabrieldonadel**
# Why

Follow up of https://github.com/expo/expo/pull/32093
Closes [ENG-13531](https://linear.app/expo/issue/ENG-13531)

# How

- update package versions
  - `react-native 0.76.0-rc.5 -> 0.76.0-rc.6`  

# Test Plan

bare-expo ios / android
fabric ios / android
ci passed

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by vonovak — CHANGES_REQUESTED** [review]
You should be able to remove 🙂 

https://github.com/expo/expo/blob/main/patches/react-native-pager-view%2B6.4.0.patch
https://github.com/expo/expo/blob/main/patches/%40stripe%2Bstripe-react-native%2B0.38.6.patch

**Comment by gabrieldonadel** [comment]
> You should be able to remove 🙂
> 
> https://github.com/expo/expo/blob/main/patches/react-native-pager-view%2B6.4.0.patch https://github.com/expo/expo/blob/main/patches/%40stripe%2Bstripe-react-native%2B0.38.6.patch

Good call @vonovak, removed

**Review by vonovak — APPROVED** [review]

File: patches/react-native-gesture-handler+2.20.0.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt b/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
+index ff440de..2e05de9 100644
+--- a/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
++++ b/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
+@@ -8,5 +8,5 @@ import com.facebook.react.uimanager.events.Event
+ 
+ fun ReactContext.dispatchEvent(event: Event<*>) {
+   val fabricUIManager = UIManagerHelper.getUIManager(this, UIManagerType.FABRIC) as FabricUIManager
+-  fabricUIManager.getEventDispatcher().dispatchEvent(event)
++  fabricUIManager.eventDispatcher.dispatchEvent(event)
```
Comment (lukmccall) [inline patches/react-native-gesture-handler+2.20.0.patch:10]: Did you raise that with the gesture handler team? If not, I can do that.

**Review by lukmccall — APPROVED** [review]

File: patches/react-native-gesture-handler+2.20.0.patch (hunk 1 above)
Comment (gabrieldonadel) [reply patches/react-native-gesture-handler+2.20.0.patch:10]: Haven't raised it yet, so feel free to contact them

**Pull request merged**